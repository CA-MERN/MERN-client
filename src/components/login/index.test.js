// import { shallow } from 'enzyme';
// import React from 'react';

// import Login from '.';

// it('renders', () => {
//     expect(shallow(<Login />)).toMatchSnapshot();
// });

// it('expects text to render', () => {
//     expect(shallow(<Login />).text()).toMatch('Log In:')
// })
