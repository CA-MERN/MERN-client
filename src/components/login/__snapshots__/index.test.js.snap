// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <h1>
    Anywhere in your app!
  </h1>
  <Formik
    initialValues={
      Object {
        "email": "",
        "password": "",
      }
    }
    onSubmit={[Function]}
    validate={[Function]}
  >
    <Component />
  </Formik>
</div>
`;
